### ISSUE TITLE
Migrate to androidx library

### ISSUE THREAD

**Issue by rkishan516** [NJACKWinterOfCode/FlutterSpace#5]
Upgrade the project to android-x.

**Comment by InfiniteVerma** [comment]
I'll like to work on this. You want me to push the changes to the android folder after migrating it to AndroidX right?

**Comment by rkishan516** [comment]
@InfiniteVerma Yes, but as you know for flutter there is a process for that given at [link](https://flutter.dev/docs/development/androidx-migration).
